components/library/openssl/openssl-1.0.2/engines/pkcs11/e_pk11.c

```diff
@@ -3707,6 +3708,15 @@ hw_aes_instruction_set_present(void)
 #if defined(__amd64) || defined(__i386)
 		present = (ui & AV_386_AES) > 0;
 #elif defined(__sparc)
+/*
+ * These symbols should already be defined from /usr/include/sys/auxv_SPARC.h
+ */
+#ifndef AV_SPARC_AES
+#define AV_SPARC_AES 0
```

Comment:
DilOS used different values for these constants. As I understand, they were coming from Solaris 11. Have you checked what Solaris 11 uses now?